app/views/content/train-to-be-a-teacher/classroom-placements.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Should the URL be `school-placements` rather than `classroom-placements`? Is there an SEO reason behind classroom?